packages/collections/graphql/resolvers/User.js

```diff
@@ -23,5 +24,22 @@ module.exports = {
     ) {
       return Collection.byNameForUser(name, user.id, context)
     }
+  },
+  async collectionItems (user, args, context) {
+    const { user: me } = context
+    if (
+      (Roles.userIsMe(user, me) && Roles.userIsInRoles(user, accessRoles)) ||
+      Roles.userIsInRoles(me, adminRoles)
+    ) {
```

Comment:
Could you DRY this code?

Something like:

```js
function mayAccess ({ user, me }) {
  return (Roles.userIsMe(user, me) && Roles.userIsInRoles(user, accessRoles)) ||
    Roles.userIsInRoles(me, adminRoles)
}
```

Reply:
Good point. I've extracted it to an helper now: https://github.com/orbiting/backends/pull/509/commits/de0a328e3de44204cd104edafe2d2204b7818365